controllers/k8sgpt_controller.go

```diff
@@ -192,6 +192,10 @@ func (r *K8sGPTReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		k8sgptNumberOfResults.Set(float64(len(response.Results)))
 		rawResults := make(map[string]corev1alpha1.Result)
 		for _, resultSpec := range response.Results {
+			// FIXME: remove this once backend is populated in the k8sgpt response
+			if resultSpec.Backend == "" {
```

Comment:
I am trying to imagine where this would happen - Would there be a simple scenario you could walk me through?

Reply:
@AlexsJones I just removed the conditional and always set backend from the k8sgptconfig. For some reason I was expecting the backend to be included in the resultSpec, but it never is.